tests/test_butlerFits.py

```diff
@@ -132,6 +140,30 @@ def assertCatalogEqual(self, inputCatalog: lsst.afw.table.SourceCatalog,
             inputRecord.getShapeErr()[2, 2],
             outputRecord.getShapeErr()[2, 2], rtol=1e-6)
 
+    def runFundamentalTypeTest(self, datasetTypeName, entity):
```

Comment:
maybe this wants a return type annotation?

Reply:
Yes, except we don't really do any type checking in obs_base yet so motivation is low since nothing I put there is checked.